src/renderer/features/governance/components/ReferendumDetails/ReferendumDetailsModal.tsx

```diff
@@ -58,16 +58,17 @@ export const ReferendumDetailsModal = ({
   const [showWalletVotes, toggleShowWalletVotes] = useToggle();
   const [showVoteHistory, toggleShowVoteHistory] = useToggle();
   const [showAdvanced, toggleShowAdvanced] = useToggle();
-  const [isModalOpen, closeModal] = useModalClose(true, onClose);
 
   const voter = useStoreMap({
     store: proposerIdentityAggregate.$proposers,
     keys: [referendum.votedByDelegate?.delegateId],
     fn: (proposers, [delegateId]) => (delegateId ? (proposers[delegateId] ?? null) : null),
   });
 
+  const closeModal = (open: boolean) => !open && onClose();
```

Comment:
Let's stick to verbose style when calling functions depending on condition

```suggestion
  const closeModal = (open: boolean) => {
    if (open) return;
    onClose();
  };
```